velox/exec/GroupingSet.cpp

```diff
@@ -1162,8 +1179,7 @@ void GroupingSet::toIntermediate(
   for (auto i = 0; i < aggregates_.size(); ++i) {
     auto& function = aggregates_[i].function;
     auto& aggregateVector = result->childAt(i + keyChannels_.size());
-    VELOX_CHECK(aggregateVector.unique());
-    aggregateVector->resize(input->size());
+    recursiveResizeChildren(aggregateVector, input->size());
```

Comment:
can you add details about this change in the description as it looks like its fixing some bug that this check discovered?
Also, should we add a test case for this fix?

Reply:
Yes this check broke this test : ReduceAggTest.fuzzGroupByAvg . And the above fix , fixes the test . The reason this test fails is that it we try and create  a dictionary wrapper on an ill formed RowVector which comes from GroupingSet::toIntermediate . This is because it sets the size of the row to input->size, but doesnt resize the children. 
Dictionary already has validations for the values vector and this is where it breaks.